modules/appeals_api/spec/fixtures/v2/valid_10182.json

```diff
@@ -21,7 +21,7 @@
         "name": "Tony Danza"
       },
       "boardReviewOption": "evidence_submission",
-      "extensionRequest": false,
+      "extensionRequest": true,
```

Comment:
Changed to text that when extension request is true, the "additional issues/pages" checkbox is checked.